chore(insights): bootstrap insights workspace and plugins (#2874)
backstage/community-plugins: workspaces/insights/plugins/insights/README.md

# insights

Welcome to the insights plugin!

> This plugin helps users gather analytics data from Backstage and show a dashboard that can help in the doption of the platform.

Reviewer: ```suggestion
This plugin helps gather user analytics data from Backstage and generates a dashboard that provides insights into the platform's adoption.
```

Author: Done.
